content/en/apps/payments/payments-agent.md

```diff
@@ -0,0 +1,341 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: The Payments Agent
+title: Learn more about the Payments Agent CorDapp
+weight: 300
+---
+
+The Payments Agent CorDapp is hosted by a trusted member of a network, such as the Business Network Operator (BNO). In this technical preview, you can assign the role of Payments Agent to a node on your local network.
```

Comment:
Check for consistency across the pages. This is how it appears on the send-payments.md: 
The Payments Agent CorDapp is hosted in a live environment by a trusted member of the network, such as the Business Network Operator (BNO). In this technical preview, you can assign the role of Payments Agent to a local node.
I think including 'live' could be misleading.